src/erk/artifacts/sync.py

```diff
@@ -73,8 +93,30 @@ def _copy_directory_contents(source_dir: Path, target_dir: Path) -> int:
     return count
 
 
+def _sync_workflows(bundled_github_dir: Path, target_workflows_dir: Path) -> int:
+    """Sync erk-managed workflows to project's .github/workflows/ directory.
+
+    Only syncs files listed in BUNDLED_WORKFLOWS registry.
+    """
+    from erk.artifacts.orphans import BUNDLED_WORKFLOWS
```

Comment:
skeptical this in needed

Reply:
Inline import is needed here: orphans.py imports get_bundled_claude_dir and get_bundled_github_dir from sync.py, so a module-level import of BUNDLED_WORKFLOWS would create a circular dependency. Added a comment explaining this.

_Addressed via `/erk:pr-address` at 2025-12-31 09:33 PST_